src/Aspire.ProjectTemplates/templates/aspire-mstest/.template.config/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    ".NET Aspire",
+    "Cloud",
+    "Web",
+    "Web API",
+    "API",
+    "Service",
+    "Test"
+  ],
+  "name": ".NET Aspire Test Project (MSTest)",
+  "defaultName": "Tests",
+  "description": "A project that contains MSTest integration tests of a .NET Aspire AppHost project.",
+  "shortName": "aspire-mstest",
+  "sourceName": "Aspire.Tests1",
+  "preferNameDirectory": true,
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "precedence": "8000",
+  "identity": "Aspire.Tests.MSTest.CSharp.8.0",
```

Comment:
Is it ok to update xunit template identity to match the same pattern? This would avoid putting one of the framework above the others.

Reply:
What do you mean by "putting one of the framework above the others"?

@phenning what are the implications of changing a template identity?

Reply:
The only impact would be if there could be two versions of the same template installed that had different identities. 

This would likely only happen if the templates were installed by different mechanism, for example, if one version was installed via an optional workload, and a second with the same group id but different identity (but identical targeted frameworks) was installed either as a result of being in the core SDK or was installed via dotnet new install. 

In that case, we would use the one with the highest precedence when that TFM was choosen. So in order to get the "newest" one, the best thing to do would be to update the precedence to say 8001 when changing identities.  

If the group identity changes, that is effectively an entirely different template, so when there were multiples installed after a rename, there would be duplicate templates in NPD.

Reply:
> What do you mean by "putting one of the framework above the others"?

We have as identity:

- xUnit: `Aspire.Tests.CSharp.8.0`
- MSTest: `Aspire.Tests.MSTest.CSharp.8.0`
- NUnit: `Aspire.Tests.NUnit.CSharp.8.0`

Which reads to me as we promote mainly xUnit and the others are supported as second class citizen. While with a rename like:

- xUnit: `Aspire.Tests.Xunit.CSharp.8.0`
- MSTest: `Aspire.Tests.MSTest.CSharp.8.0`
- NUnit: `Aspire.Tests.NUnit.CSharp.8.0`

We have the different test frameworks on the same "level"

Reply:
Ping @DamianEdwards

Reply:
I see. These identities aren't actually visible to the user but making them consistent makes sense if there's no downside. I think we actually want to update the group identity too so this will effectively EOL the current xUnit template when we release this change, but I don't think that's a problem.

Reply:
I'll do the change as part of this PR but please let me know if you would prefer to have this one merged and another one for the xUnit template change.

Also, shall I go ahead and also provide NUnit?

Reply:
Done for both points.

cc @OsirisTerje